server/src/models/group.model.ts

```diff
@@ -6,8 +6,8 @@ const groupSchema = new Schema<GroupSchema>(
     groupName: { type: String, required: true },
     userIds: [{ type: Types.ObjectId, ref: 'User' }],
     setlistIds: [{ type: Types.ObjectId, ref: 'Setlist' }],
-    createdBy: { type: String },
-    lastUpdatedBy: { type: String },
+    createdBy: { type: Schema.Types.ObjectId, ref: 'User' },
```

Comment:
Types.ObjectId is enough right? since you imported Types haha

Reply:
actually cannot, i tried lol. somehow its different

Reply:
I got the same error last time. Its fixed if u change the createdBy in types file to just ObjectId. Idk why if its non array it has to be like this lol.

Reply:
@Midnite1212 lol so i searched and
> Schemas should always use mongoose.Schema.Types.
> 
> mongoose.Types are the object you work with within a mongoose document. These are special array subclasses, 
> buffer subclasses, etc that we've hooked into or created special to ease updates and track changes.

 https://stackoverflow.com/questions/28997636/should-i-use-schema-types-objectid-or-schema-objectid-when-defining-a-mongoose-s

Reply:
Well always better to follow convention haha. Will change mine to Schema.types :)